Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -53,6 +54,84 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
             });
         },
 
+        /**
+         * Fetches data about user
+         *
+         * @method _fetchUserInfo
+         * @param event {Object} event facade
+         */
+        _fetchUserInfo: function (event) {
+            var user = this.get('app').get('user'),
+                userService = this.get('capi').getUserService(),
+                userInfo = {},
+                promise = new Y.Promise(function (resolve, reject) {
+                    userService.loadUser(user.get('id'), Y.bind(function (error, response) {
+                        try {
+                            userInfo.userData = response.document.User;
+                            resolve(user.get('id'));
+                        } catch (e) {
+                            reject({
+                                message: 'Cannot get user data',
+                                identifier:  'fetch-user-info-' + Date.now()
+                            }, e);
+                        }
+                    }, this));
+                });
+
+            promise.then(function(userId) {
+                return new Y.Promise(function (resolve, reject) {
+                    userService.getRoleAssignmentsForUser(userId, Y.bind(function (error, response) {
+                        try {
+                            if (response.document.RoleAssignmentList && response.document.RoleAssignmentList.RoleAssignment[0]) {
+                                resolve(response.document.RoleAssignmentList.RoleAssignment[0].Role._href);
```

Comment:
I'm good with your recommendation, Damien. It's a nice-to-have feature, but not essential.